ptx/sec_work.ptx

```diff
@@ -523,7 +530,14 @@
           <caption>Illustrating a water tank in order to compute the work required to empty it in <xref ref="ex_pump1">Example</xref></caption>
           <!-- START figures/fig_work_pump1.tex -->
           <image xml:id="img_pump1a" width="47%">
-            <description></description>
+            <description>
+              Image of a cylindrical storage tank with a radius of 10ft and a height of 30ft.
```

Comment:
Add a space between the number and the unit. Even better: use the built-in markup for quantities:
`<quantity><mag>10</mag><unit base="foot"/></quantity>`

Reply:
Same comment applies for further descriptions below, but I'll avoid repeating it each time.